src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -435,12 +443,10 @@ class LibraryServiceClient
         }
 
         $headerDescriptor = new AgentHeaderDescriptor([
-            'clientName' => $options['appName'],
-            'clientVersion' => $options['appVersion'],
+            'libName' => $options['libName'],
+            'libVersion' => $options['libVersion'],
             'codeGenName' => self::CODEGEN_NAME,
```

Comment:
`'codeGenName'` should just be hard-coded as `'gapic'` in GAX. We should not leave it open-ended.